.travis.yml

```diff
@@ -17,17 +17,29 @@
 # under the License.
 
 # .travis.yml
-# For maven builds
-language: java
+language:
+  # For maven builds
+  - java
+  # For web ui build
+  - node_js
 dist: trusty
+<<<<<<< HEAD
```

Comment:
This must have gotten in there by mistake

Reply:
This is preventing the Travis CI to check this PR. Can this be fixed? I think just replacing `sudo:false` with `sudo:required` in the original code would be sufficient. @skystar-p or @seojangho ? I don't have access to the original branch.